### PR TITLE
[sil_cipher_1.1] Changed CTRL to RALT, added toggle key for jianpu, u…

### PR DESCRIPTION
**Pull request by LornaSIL**
…pdated docs for changes

Addresses issues in #947

### CODE REVIEW

**Comment by DavidLRowe** [comment]
README.md should have "Version 1.1"

Here's something I tried:
- created a .kmp and installed it on my machine
- used the keyboard options to select Chinese
- opened Notepad
- typed RALT+= and Notepad crashed

Not sure what I did wrong.

**Comment by LornaSIL** [comment]
Oh, you shouldn't use both the ways to switch keyboards. You should only do one or the other. I guess I need to add that to the documentation. But, it shouldn't crash.

**Comment by LornaSIL** [comment]
Is there a reason README.md needs the version? I removed it so I don't have to update that file every time.

File: release/sil/sil_cipher_music/source/welcome/welcome.htm [hunk 1]
```diff
@@ -22,48 +22,53 @@
...
+line-spacing to a fixed height.</p>
+<p>The
+font targets Indonesian cipher (<i>kepatihan</i>), but a mode
+for Chinese cipher (<i>jianpu</i>)
 is also provided via the &quot;slur&quot; font feature. The purpose
-and scope of this project, and downloads, are provided here:</span></span></font></p>
+and scope of this project, and downloads, are provided here:</p>
 <p style="margin-top: 0.08in; margin-bottom: 0.08in; line-height: 0.2in">
 <span style="background: transparent"> 
 <font face="Doulos SIL Cipher"><span style="font-style: normal"><a class="western" href="https://software.sil.org/doulos-sil-cipher/">https://software.sil.org/doulos-sil-cipher/</a></span></span></font></p>
```
Comment (darcywong00) [inline release/sil/sil_cipher_music/source/welcome/welcome.htm]: Separate from this PR, but you can also add sil_cipher_music to the "Keyboard" section of 
https://software.sil.org/doulos-sil-cipher/

File: release/sil/sil_cipher_music/sil_cipher_music.keyboard_info [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+	  "description": "This keyboard is for typing Cipher music notation (where numbers represent notes). The Cipher music notation is used throughout Indonesia and China for all kinds of music. The keyboard is not intended for general orthographic use, although it is usable for song lyrics in simple Roman writing systems. This solution is primarily intended for fairly straightforward songbooks for singers, not for complex instrumental music. It can handle harmony lines fine, but not a lot of intricacies or subtleties (e.g. no staccato). The default for this keyboard is Indonesian kepatihan. The keyboard has an option key to change to Chinese jianpu. Please read the documentation for how to switch to Chinese jianpu."
```
Comment (darcywong00) [inline release/sil/sil_cipher_music/sil_cipher_music.keyboard_info]: Depending how soon keymanapp/keyman#2334 gets 🍒-picked and kmcomp updated in this repo, you might want to ensure we don't include mobile platforms:
```
    "platformSupport": {
        "windows": "full",
        "macos": "full",
        "linux": "full",
        "desktopWeb": "full"
   }
```

File: release/sil/sil_cipher_music/source/sil_cipher_music.kvks [hunk 3]
```diff
@@ -7,10 +7,10 @@
       <usealtgr/>
     </flags>
   </header>
-  <encoding name="unicode" fontname="Doulos SIL Cipher" fontsize="12">
-    <layer shift="SRC">
-      <key vkey="K_PERIOD">̣</key>
-      <key vkey="K_COMMA">̇</key>
+  <encoding name="unicode" fontname="Segoe UI Symbol" fontsize="12">
```
Comment (darcywong00) [inline release/sil/sil_cipher_music/source/sil_cipher_music.kvks]: Was this intentional not to use Doulos SIL Cipher?

File: release/sil/sil_cipher_music/source/welcome/welcome.htm (hunk 1 above)
Comment (LornaSIL) [reply release/sil/sil_cipher_music/source/welcome/welcome.htm]: >Separate from this PR, but you can also add sil_cipher_music to the "Keyboard" section of
https://software.sil.org/doulos-sil-cipher/

It was there, but not very obvious. I've made it more prominent in both the Keyboard section and in the download for InKey section.

File: release/sil/sil_cipher_music/sil_cipher_music.keyboard_info (hunk 2 above)
Comment (darcywong00) [reply release/sil/sil_cipher_music/sil_cipher_music.keyboard_info]: #962 updated kmcomp to 12.0.55.0

If you merge in master, you should be able to undo this change

Comment (LornaSIL) [reply release/sil/sil_cipher_music/sil_cipher_music.keyboard_info]: Done.

File: release/sil/sil_cipher_music/source/sil_cipher_music.kvks (hunk 3 above)
Comment (LornaSIL) [reply release/sil/sil_cipher_music/source/sil_cipher_music.kvks]: Changed back to the Doulos Cipher font. That was an unintentional change.

**Pull request merged**